discovery-provider/alembic/versions/90ad232bfe92_triggers_count_from_scratch.py

```diff
@@ -0,0 +1,52 @@
+"""triggers count from scratch
+
+Revision ID: 90ad232bfe92
+Revises: ab56e2d974a6
+Create Date: 2022-08-02 17:21:16.257167
+
+"""
+from pathlib import Path
+
+import sqlalchemy
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "90ad232bfe92"
+down_revision = "ab56e2d974a6"
+branch_labels = None
+depends_on = None
+
+
+def load_sql(name):
```

Comment:
would be nice to separate this out to be re-used. I believe this is already defined in another migation